version.txt

```diff
@@ -1,2 +1,2 @@
-Plone version: 5.2.10.2 (with Zope 4.8.1)
-Installer version: Unified Installer 5.2.10.2-r1
+Plone version: 5.2.11 (with Zope 4.8.1)
+Installer version: Unified Installer 5.2.11-r1
```

Comment:
@mauritsvanrees you might want to remove the `-r1` 🍀

Reply:
It has been in there for years, since [5.2.0-r1](b66e35b16e4ebd91a60ad16dd10b064c274935fe). I guess it was meant as only needed when there is a release and later a revision of the release.
I guess it does not matter: the version is not used anywhere.